src/main/kotlin/io/github/vootelerotov/renovator/Renovator.kt

```diff
@@ -81,48 +80,90 @@ class Renovator : CliktCommand() {
 
   private fun merge(pr: PullRequest, pullRequestClient: PullRequestClient, confirmedViaFlag: Boolean = false) {
     if (pr.merged() != false) {
-      echo("PR ${prDescription(pr)} is already merged")
+      echo("PR with ${prDescriptionShort(pr)} is already merged")
       return
     }
 
     if (pr.mergeable().getOrNull() != true) {
-      echo("PR ${prDescription(pr)} cannot be merged")
+      echo("PR with ${prDescriptionShort(pr)} cannot be merged")
       return
     }
 
-    val isConfirmed = confirmedViaFlag || confirm("Approve and merge PR ${prDescription(pr)}?")
-      ?: throw IllegalStateException("Can not get confirmation from stdin")
+    val isConfirmed = confirmedViaFlag || confirmMerge(pr)
 
     if (!isConfirmed) {
       return
     }
 
-    val comment = prompt("Enter comment to approve the PR with", default = defaultComment)
+    val comment = if (defaultComment.isNullOrBlank()) {
+      prompt("Enter comment to approve the PR with")
+    } else {
+      prompt(
+        "Press enter to approve the PR with the default comment '$defaultComment' or enter a different comment to approve PR with",
+        default = defaultComment,
+        showDefault = false
+      )
+    }
       ?: throw IllegalStateException("Can not get comment from stdin")
 
-    echo("Approving PR ${prDescription(pr)} with comment $comment")
+    echo("Approving PR ${pr.title()} with comment $comment")
```

Comment:
Good changes here and on line 116. Down the line, I guess `-d, --debug` flag and adding them behind them would be good. Honestly, main reason why they are here was because I wanted to see if the stuff I was writing was working :sweat_smile: